server/jetstream_benchmark_test.go

```diff
@@ -307,17 +307,20 @@ func BenchmarkJetStreamConsume(b *testing.B) {
 							if verbose {
 								b.Logf("Setting up %d nodes", bc.clusterSize)
 							}
-							var connectURL string
+							var (
+								connectURL string
+								cl         *cluster
```

Comment:
Personally don't like the grouping of vars, but it's matter of personal preference